pkg/scanner/local/scan.go

```diff
@@ -58,19 +59,25 @@ type OspkgDetector interface {
 	Detect(imageName, osFamily, osName string, repo *ftypes.Repository, created time.Time, pkgs []ftypes.Package) (detectedVulns []types.DetectedVulnerability, eosl bool, err error)
 }
 
+type ScannerOptions struct {
```

Comment:
Won't it be better to have a scan type attribute or a method in the scanner interface which  provides the scan type? I believe ArtifactType already exists. Won't it solve the purpose?

Reply:
Addressed